docs/source/quickstart/99_FAQ.md

```diff
@@ -59,3 +59,14 @@ You can review the logs to see the error with:
 ```Bash
 oc logs -n openstack -l component=cinder-api
 ```
+
+## Working in air-gapped environment
+
```

Comment:
We can follow [shellcheck guide](https://ci-framework.readthedocs.io/en/latest/internals/02_testing.html#spellchecking) to fix shellecheck job